### PR TITLE
Upgrade onionwrapper to 0.0.5

### PR DESCRIPTION
**Pull request by akwizgran**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -120,7 +120,7 @@ dependencies {
     kapt "com.google.dagger:hilt-compiler:$hilt_version"
 
     // Tor
-    implementation "org.briarproject:onionwrapper-android:0.0.4"
+    implementation "org.briarproject:onionwrapper-android:0.0.5"
     tor 'org.briarproject:tor-android:0.4.7.13-2'
```
Comment (grote) [inline app/build.gradle:124]: Can/should we update tor to 0.4.7.14 at the same time?

**Review by grote — APPROVED** [review]

File: app/build.gradle (hunk 1 above)
Comment (grote) [reply app/build.gradle:124]: Ah looks like we haven't released that yet for some reason. Considering the onion services issue, we probably should.

**Comment by grote** [comment]
GitHub is complaining:

> The base branch requires all commits to be signed.

File: app/build.gradle (hunk 1 above)
Comment (akwizgran) [reply app/build.gradle:124]: Yup, there are some extra Docker hoops to jump through for the Mac binaries but I should be able to upload them tomorrow. I'll leave this open till then and add a commit to upgrade the Tor binaries.

**Comment by akwizgran** [comment]
> The base branch requires all commits to be signed.

Ah sorry, I've amended the commit and force-pushed.

**Pull request merged**